packages/flutter/lib/material.dart

```diff
@@ -10,14 +10,10 @@
 ///
 /// See also:
 ///
-///  * [flutter.dev/widgets/material](https://flutter.dev/widgets/material)
+///  * [docs.flutter.dev/ui/widgets/material](https://docs.flutter.dev/ui/widgets/material)
 ///    for a catalog of commonly-used Material component widgets.
-///  * [material.io/design](https://material.io/design/)
+///  * [m3.material.io](https://m3.material.io/)
```

Comment:
```suggestion
///  * [m3.material.io](https://m3.material.io/) for the Material 3 specification
///  * [m2.material.io](https://m2.material.io/) for the Material 2 specification
```

Reply:
@guidezpl Fixed https://github.com/flutter/flutter/pull/129891/commits/b3b435221dbce09e9d302db9529f5c2d47d9b961 and https://github.com/flutter/flutter/pull/129891/commits/535d3c8932ab301c92c52338ea69805559944bc4